src/components/Profile/MyPosts/MyPosts.jsx

```diff
@@ -1,48 +1,27 @@
 import React from 'react';
+import {Field, reduxForm} from 'redux-form';
+
 import s from './MyPosts.module.css';
 import Post from './Post/Post';
-import { Field, reduxForm } from 'redux-form';
-import { required, maxLengthCreator } from '../../../utils/validators/validators';
-import { Textarea } from '../../common/FormsControl/FormsControls';
-
-const MyPosts = React.memo(props => {
-  let postsElements = props.posts.map(p => <Post message={p.message} likesCount={p.likesCount} />);
+import {required, maxLengthCreator} from '../../../utils/validators/validators';
+import {Textarea} from '../../common/FormsControl/FormsControls';
 
-  let newPostElement = React.createRef();
+const MyPosts = props => {
+  let postsElements = props.posts.map(p => <Post key={p.message} message={p.message} likesCount={p.likesCount} />);
 
   let onAddPost = values => {
     props.addPost(values.newPostText);
   };
 
-  //ojoon
   return (
     <div className={s.postsBlock}>
       <h3>My posts</h3>
-      {/* Redux-form
-      <form>
-        <div>
-          <div>New post</div>
-        </div>
-        <textarea
-          onChange={onPostChange}
-          ref={newPostElement}
-          value={props.newPostText}
-        />
-      </form>
-      <div>
-        <button onClick={onAddPost}>Add post</button>
-        <button>Remove</button>
-      </div> */}
       <AddNewPostFormRedux onSubmit={onAddPost} />
 
-      <div className={s.posts}>
-        {postsElements}
-        {/* <Post message= {postData[0].message} likesCount={postData[0].likesCount} />
-        <Post message= {postData[1].message} likesCount={postData[1].likesCount}/> */}
-      </div>
+      <div className={s.posts}>{postsElements}</div>
```

Comment:
no need to move `props.posts.map(...)` into a separate variable. `props.posts.map(...)` can be used right here